### PR TITLE
Conversions: Fix #3534

### LINKED ISSUE #3534: Conversions: Can't convert various metric measurements
I can't convert to or from Hecto-, Deka-, Deci-, and get an instant answer. Also, Mega- gets counted as Milli- (Mega uses M and Milli uses m). There may be more.

------
IA Page:  http://duck.co/ia/view/conversions 
[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @mintsoft

### PR DESCRIPTION
**Pull request by holtjohnson**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [ ] New Instant Answer
    - [ ] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [x] Improvement
    - [x] Bug fix: **`Conversions: Fix #3534`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

I added some more metric measurements and sorted them by size.

###### Which issues (if any) does this fix?

Fixes #3534 - Add additional metric measurements.

###### People to notify (@mention interested parties)
@mintsoft 
@Zekario

---

Instant Answer Page: https://duck.co/ia/view/conversions

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @mintsoft

### CODE REVIEW

**Comment by daxtheduck** [comment]
@holtjohnson Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/conversions/ratios.yml [hunk 1]
```diff
@@ -432,6 +528,42 @@ unit: litre
...
+unit: hectolitre
+plural: hectolitres
+---
+aliases:
+  - decaiter
+  - decaiters
+  - dal
+factor: 0.1
+type: volume
+unit: decaitre
```
Comment (mintsoft) [inline share/goodie/conversions/ratios.yml]: I think the SI name of this unit is `decalitre` and the plural `decalitres` ?

**Comment by mintsoft** [comment]
Thanks for doing this @holtjohnson It should definitely plug some of the query holes. I've just checked over it and it all looks OK with 1 exception; I think there's a typo in the decalitre name. Other than that it looks ready to go

**Comment by holtjohnson** [comment]
You're correct. I fixed the typo @mintsoft.

**Comment by mintsoft** [comment]
Awesome. Muchos gracias amigo

**Pull request merged**